static/styles/components.css

```diff
@@ -160,11 +160,22 @@
     width: 90px;
     margin: 0px -0.5px;
     text-align: center;
+    text-overflow: ellipsis;
+    white-space: nowrap;
+    overflow: hidden;
+    vertical-align: bottom; /* See http://stackoverflow.com/a/43266155/ */
     padding: 3px 10px;
     background-color: #fff;
     cursor: pointer;
 }
 
```

Comment:
Ah yeah sorry I mistyped it. Those styles apply to the `.tab-switcher .ind-tab` and you put `display: flex` on the `.tab-switcher`.